manifests/l/Luxafor/Software/1.0.0/Luxafor.Software.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=NVS0.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Luxafor.Software
+PackageVersion: 1.0.0
+InstallerLocale: en-US
+InstallerType: zip
+Scope: machine
+InstallerSwitches:
+  Silent: --silent
+  SilentWithProgress: -silent
+UpgradeBehavior: install
+Protocols:
+- https
+ReleaseDate: 2023-06-21
+Installers:
+- Architecture: x64
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: ~/desktop/installedpackages.json
+  InstallerUrl: https://www.luxaformanual.com/_files/archives/b41627_031bcebfeed64d47863e8d3605536f81.zip?dn=luxafor_win_v2.1.5.26-1.zip
+  InstallerSha256: 0FA06FF7E673FAABF40DC96CE835759F17376F2DA81FF7C0524E83CB271F1059
+  ProductCode: luxafor pomodoro timer
+- Architecture: x64
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: ~/desktop/installedpackages.json
+  InstallerUrl: https://www.luxaformanual.com/_files/archives/b41627_36d3a39866cf47cbbe909ed3b9e8f0e1.zip?dn=luxafor-mute-setup-2022-04-24.zip
+  InstallerSha256: A3E81F693F944B23106F63E3CB680E5F3472D115F8F8A6E3A6645D5C338C3E8B
```

Comment:
This error is due to having no differentiating settings between the 2 installers. The second installer entry seems to be for a different package, based on the file name in the InstallerUrl.